### PR TITLE
MGDOBR-973: Notify UI team if there is a change in the manager api or the json schemas

### PR DESCRIPTION
**Pull request by r00ta**
[MGDOBR-973](https://issues.redhat.com/browse/MGDOBR-973)

## PR owner checklist

Please make sure that your PR meets the following requirements:

- [ ] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config).  
  *Run `mvn clean verify -DskipTests` so that imports are correctly set.*
- [ ] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-1] - $clear_explanation_of_what_you_did"
- [ ] The layers in the `kustomize` folder have been updated accordingly.
- [ ] All new functionality is tested
- [ ] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

## PR reviewer(s) checklist

- [ ] Check if code and Github action workflows have been modified and if the modification is safe
- [ ] If the modification is safe, add the `safe to test` label

<details>
<summary>
How to trigger pipelines and use the bots:
</summary>

- <b>Run the end to end pipeline</b>  
  Annotate the pull request with the label: `safe to test`. If you want to run the pipeline again, remove and add it again.

- <b>Rebase the pull request</b>  
  Comment with: `/rebase`.
- <b>Deploy BOT</b>
  Comment with `/deploy <dev,stable> when the PR has been merged to deploy to a target environment.

</details>

### CODE REVIEW

File: .github/workflows/check-and-assign-pr-to-ui-team.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+      - 'processor/configuration/src/main/resources/schemas/*'
+      - 'openapi/smartevents_mgmt.yaml'
+jobs:
+  assign:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: delivery-much/actions-assigner@v1
+        with:
+          token: ${{ secrets.GH_TOKEN }}
+          reviewers: vpellegrino,kelvah
```
Comment (jomarko) [inline .github/workflows/check-and-assign-pr-to-ui-team.yml:16]: could we please include also QE? me @sjamboro  or @ChughShilpa ?

**Review by jomarko — COMMENTED** [review]
one comment

File: .github/workflows/check-and-assign-pr-to-ui-team.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+name: Check and Assign PR to UI team
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+    paths:
+      # assign to UI team only if there is a change in a json schema/catalog or in the manager API
+      - 'processor/configuration/src/main/resources/schemas/*'
+      - 'openapi/smartevents_mgmt.yaml'
```
Comment (manstis) [inline .github/workflows/check-and-assign-pr-to-ui-team.yml:8]: The UI repository already has this check (but in [reverse](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/blob/main/.github/workflows/openapi-sdk-compatibility-check.yml)).

Comment (manstis) [reply .github/workflows/check-and-assign-pr-to-ui-team.yml:8]: But this does no harm either :smile:

**Review by manstis — APPROVED** [review]

File: .github/workflows/check-and-assign-pr-to-ui-team.yml (hunk 2 above)
Comment (r00ta) [reply .github/workflows/check-and-assign-pr-to-ui-team.yml:8]: IIUC with that check the UI team is "notified" about the changes only after we have merged something. Watching the openapi file here we ensure that they are always notified if any change is going to be applied in the API, which is good

File: .github/workflows/check-and-assign-pr-to-ui-team.yml (hunk 1 above)
Comment (r00ta) [reply .github/workflows/check-and-assign-pr-to-ui-team.yml]: I've added you to the team. Please accept the invitation

Comment (jomarko) [reply .github/workflows/check-and-assign-pr-to-ui-team.yml]: @r00ta I do not see a notification in email or github, where should I approve it please?

**Review by danielezonca — APPROVED** [review]

**Pull request merged**